src/react-components/checkbox.tsx

```diff
@@ -26,7 +26,7 @@ export const Checkbox = ({
   dataTestid,
   ...props
 }: CheckboxProps): JSX.Element => {
-  const id = Math.random().toString(36).substring(2)
+  const id = props.id ?? Math.random().toString(36).substring(2)
```

Comment:
This way things still work when we pass an id (as we do for the "remember me" checkbox).

Reply:
makes sense :+1: